### PR TITLE
Fixes wrong diagnostic range for missing attribute quotations

### LINKED ISSUE #135: Wrong diagnostic range for malformed attribute
Given 
```xml
<?xml version="1.0" encoding="UTF-8"?>
<foo>
  <bar  one= two="" />
</foo>
```
the error range for `Open quote is expected for attribute "one" associated with an  element type  "bar".`  is underlying `two`, instead of `one`

### PR DESCRIPTION
**Pull request by NikolasKomonen**
Fixes #135

### CODE REVIEW

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/contentmodel/participants/XMLSyntaxErrorCode.java [hunk 1]
```diff
@@ -148,8 +148,7 @@ public static Range toLSPRange(XMLLocator location, XMLSyntaxErrorCode code, Obj
 		case NameRequiredInReference:
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/contentmodel/participants/XMLSyntaxErrorCode.java]: should break here I think

**Comment by NikolasKomonen** [comment]
@fbricon Fixed

**Comment by angelozerr** [comment]
@NikolasKomonen it should be cool if you could add a code action to add those missing quotes lFor that, see existing code action at https://github.com/angelozerr/lsp4xml/tree/master/org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/contentmodel/participants/codeactions

**Comment by fbricon** [comment]
@angelozerr please open an enhancement request

**Comment by angelozerr** [comment]
> @angelozerr please open an enhancement request

Done https://github.com/angelozerr/lsp4xml/issues/137

**Pull request merged**